bin/figaro

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env ruby
+# frozen_string_literal: true
+#
+# This file was generated by Bundler.
+#
+# The application 'figaro' is installed as part of a gem, and
+# this file is here to facilitate running it.
+#
+
+require "pathname"
```

Comment:
I was following the slides.

It may have occurred due to this:

bundle binstubs figaro